1.1.0/shipment-status.md

```diff
@@ -852,7 +852,7 @@ If all goes well, the company **Corp** will receive a response like this:
   "shipments": [
     {
       "id": "d4fd1f2c-642f-4df8-a7b3-139cf9d63d17",
-      "shipmentNumber": "SHP001",
```

Comment:
I am not sure I understand this change from the value `SHP001` to `SU001SHP001` for the `shipmentNumber` for the responses coming from the _Pulp_ company (_Supplier_). I think we should discuss it during our next meeting...